aqt/settings.ini

```diff
@@ -23,22 +23,23 @@ blacklist:
     http://mirrors.geekpie.club
 fallbacks:
     https://qtproject.mirror.liquidtelecom.com/
+    https://mirrors.aliyun.com/qt/
     https://mirrors.sjtug.sjtu.edu.cn/qt/
+    https://mirrors.tuna.tsinghua.edu.cn/qt/
```

Comment:
This is problematic because it limit access connections unknown condition, maybe outside of china, and randomly failed, So it is listed in black list and it had been failed in past CI tests.
Could you remove blacklisted host?

Reply:
Or if  it is healthy, please also update blacklist.

Reply:
Thanks for the pointer, should have caught that myself. I've updated the PR to avoid adding blacklisted entries.
While the mirror is currently reachable for me, it's rather slow and I don't know the history, so it might be better to keep it as-is.